pkgs/build-support/build-fhs-userenv-bubblewrap/env.nix

```diff
@@ -38,9 +38,10 @@ let
   # these match the host's architecture, glibc_multi is used for multilib
   # builds. glibcLocales must be before glibc or glibc_multi as otherwiese
   # the wrong LOCALE_ARCHIVE will be used where only C.UTF-8 is available.
+  glibc = with pkgs; (if isMultiBuild then glibc_multi else glibc);
```

Comment:
I think the infinit recursion is occurring here. Can the pkgs be removed and glibc added directly?